Resources/Textures/_NF/Objects/Misc/module.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from shiptest at https://github.com/shiptest-ss13/Shiptest/pull/1473 | resprited by erhardsteinhauer (discord/github)",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "hacked-merc-fab-board"
+    }
+  ]
+}
```

Comment:
Nice, should move the NFSD stuff in here at some point, since the original's metadata file's a mess.